### PR TITLE
Release 1.0 and update install docs

### PR DESCRIPTION
**Pull request by wagner-certat**
Scream now if I forgot something.

As soon as someone reviews this, I will push the tag and the packages.

### CODE REVIEW

**Comment by wagner-certat** [comment]
A preview for the packages can be found in the unstable repository: https://build.opensuse.org/package/show/home:sebix:intelmq:unstable/intelmq

File: intelmq/tests/bots/experts/cymru_whois/test_expert.py [hunk 1]
```diff
@@ -71,8 +71,8 @@
 EXAMPLE_6TO4_OUTPUT = {"__type": "Event",
                   "source.ip": "2002:3ee0:3972:0001::1",
                   "source.network": "2002::/16",
-                  "source.asn": 6939,
-                  "source.as_name": "HURRICANE - Hurricane Electric, Inc., US",
+                  "source.asn": 1103,
```
Comment (aaronkaplan) [inline intelmq/tests/bots/experts/cymru_whois/test_expert.py]: note: might change in the future. Anyways...

**Comment by wagner-certat** [comment]
@aaronkaplan please approve the changes, thanks!

**Comment by aaronkaplan** [comment]
yes, let's go!

**Pull request merged**